content/knowledge/scalinghistory.md

```diff
@@ -0,0 +1,132 @@
+   As long as the majority of nodes are not malicious, this will ensure security of the system. 
+   Complex web of overlapping quorum slices makes it almost impossible for the majority of nodes to collude to control consensus. 
+   FBA can be used on top of PoW or PoS (to be discussed later). 
+   Safety and fault tolerance are [preferred over liveness](https://www.youtube.com/watch?v=aU08km2xrz0&ab_channel=Lumenauts). 
+   In case of an accidental fork, the system is halted until consensus is reached. 
+   This is important in case of banking applications. 
+   Consensus that requires only message passing followed by voting process leads to high transaction volume per second and less expenditure of electricity than PoW.
+   However, a criticism of pure FBA is that it leads to fragile structures of constituent nodes, permitting topology scenarios where a single node failure can cause a network-wide failure
+   (Resistant to Forks?)
+
+   TODO: Drawback: Single point failure if quorum sets share a single node?
+   (Not mentioning about BFT which require a pre recommended validator list as it is against decentralisation)
+
+## 3. Proof of Stake (PoS)
+
+   Then came the Proof of Stake(PoS) consensus algorithms (e.g. Casper). 
+   This no longer needs the computational complexity for validation of blocks but rather how much a node or node pool has staked and how long it has been active. 
+   Staking is like putting some native cryptocurrency as a collateral. 
+   The higher the stake and time in the game, the higher the chances of getting to write a block on behalf of the whole ecosystem and in return receive some cryptocurrency. 
+   This can be distributed among the participating nodes in case of a node pool where there are multiple parties participating by delegating their tokens. 
+   Two keywords are different here compared to PoW: (1) Computers that participate in PoS consensus are called Validators, (2) Once a block has been created and accepted by the network, it is said to be forged (not mined). 
+   Then the validator gets some reward. 
+   If a validator tries to be malicious then it gets penalised losing some of its staked cryptocurrency.
+   PoS is much faster than PoW as there is no need to solve complex cryptographic puzzles to get to be the block writer and the process is much more streamlined. 
+   Also no unnecessary wastage of energy means PoS is more environment friendly.
+   The security of the network is ensured by the fact that if an attacker were to attack the system, it needs to have 51% of all the value staked. 
+   As opposed to 51% of all computing power in case of a PoW system. 
+   Sustainability, security are achieved in a PoS model like the one in ethereum2.0 which is on its way to be released.
+   PoS ensures more throughput compared to PoW model. 
+   However there is a communication overhead which hinders scalability (TODO: Explain more). 
```

Comment:
How exactly the communication overhead is present in case of PoS? A full mechanism example would be useful as this is not completely clear to me. The example can include the entire process from a user broadcasting a transaction to the validators actually getting rewarded so that the overload of communication can be visualised by the reader.